modules/accredited_representative_portal/db/migrate/20250128162530_create_ar_power_of_attorney_form_submissions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArPowerOfAttorneyFormSubmissions < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_power_of_attorney_form_submissions do |t|
+      t.uuid :power_of_attorney_request_id, null: false
+      t.string :service_id
```

Comment:
I wonder if there is value to the DB being set up with a constraint that `service_id` should be unique among our form submission records?